packages/wdio-utils/src/envDetector.js

```diff
@@ -38,31 +38,30 @@ function isW3C (capabilities) {
 }
 
 /**
- * check if session is run by Chromedriver
- * @param  {Object}  capabilities  caps of session response
- * @return {Boolean}               true if run by Chromedriver
+ * check if session is run by a chromium based driver (chromedriver, msedgedriver)
+ * @param  {Object}  capabilities  capabilities of session response
+ * @return {Boolean}               true if run by chromium based driver
  */
-function isChrome (caps) {
```

Comment:
some places used "caps" and others "capabilities".  trivial but, cleaned up so its consistent and easier to read